SearchTests/ResponseParsingTests.swift

```diff
@@ -23,6 +23,59 @@ class ResponseParsingTests: XCTestCase {
         }
     }
 
+    private func validateResponse(response: Response) {
+        validateHeader(response.header)
+        validateResults(response.results)
+    }
+
+    private func validateHeader(header: Header) {
+        expect(header.uri).to(equal("https://api.ordnancesurvey.co.uk/places/v1/addresses/find?query=Ordnance%20Survey%2C%20Adanac%20Drive%2C%20SO16&maxresults=1"))
```

Comment:
Interesting there's a lot of extra data in the header for search. Might require a little thought to use the stuff I've extracted to the base framework, but we can either subclass header or use some kind of extension...